src/api/donors/donors.resolver.ts

```diff
@@ -22,6 +23,21 @@ export class DonorsResolver {
     return this.donorsService.findOne(id);
   }
 
+  @Query(() => DonorPagination, { name: 'donorsByOrganization' })
```

Comment:
I don't think we need these ones. Remember, donors are only associated with donations, not separately.